ACMS-3478: Update README.md file. (#27)
acquia/drupal-recommended-settings: README.md

and deployable on Acquia Cloud.

## Installation
###Install using Composer 

```
composer require acquia/drupal-recommended-settings
```
### Include Acquia Global Command in DRS for multisite functionality

> To setup multi-site with Drupal Recommended Settings, please add below section in

Reviewer: Remove below section from readme file. as it will be moved to github/acquia from github/vishalkhode1, once it's been moved to github/acquia we can release a dev version.

$dbSpec = [
  'drupal' => [
    'db' => [

>       'database' => 'drupal', // In case of multi-site database name is replaced with the site name.

Reviewer: ```suggestion
      'database' => $siteUri, //In the event of a database name change, the name should be replaced accordingly.
```
